modtools/create-unit.lua

```diff
@@ -25,7 +25,6 @@ local usage = [====[
 modtools/create-unit
 ====================
 Creates a unit.  Usage::
-
```

Comment:
yeah ignore those if you want those lines.   I was having an issue where travis kept saying I had some funky indentation.... I can't find anything else in the file... was going to move a few lines and see if travis saw it differently.